### DISCUSSION TITLE
Convert v5 board to support remote power like v6 with jumper off?

### DISCUSSION THREAD

**Discussion by benjamink** [tjboldt/Apple2-IO-RPi#117] [General]
Just received my v5 board (it's what you had left that was built/tested) & I'm wondering if it could be converted to support remote power easily?  Seems like it's just a large trace that provides the power to the RPi that would have to be cut & a bodge wire added to re-enable on-board power again.  I'm thinking this could be run to a switch or something if I wanted.  Is that all it'd take?

**Comment by tjboldt** [comment]
Yes, that would work.

![v5-cut](https://user-images.githubusercontent.com/21348575/213902870-4eacc18f-951e-4ff5-9973-ac21745eae56.jpg)

**Comment by OrangeBox72** [comment]
Mr. Boldt.. please see below.

Here is what I am doing with my Prototype-Version-5 **Apple2-IO-RPI board**.

(Just received the last of my long overdue chips.. aka chip-shortage / 1.5yr wait).
However, I still don't have a **Raspberry Pi Zero**!

So, I am going to try a **Raspberry Pi 3 B 1.2** using a ribbon cable.. (a cable that I will cut **strands 2,4** aka 5V).
Since the ribbon cable strands are on the end of the cable, I am going to cut **strand 1** aka 3V which is NOT used by TJBoldt's board.

My question to Mr. Boldt is... 
It would make cutting the ribbon-cable very easy if I could cut strands 1, 2, 3, and 4.
My failing eyes seem to believe that you do not use PIN-3 (GPIO2), but I would love to get your feedback before I get out my Xacto Knife.

NOTE: Strands 1,2,3,4 colors... edge - brown, red, orange yellow.



![IMG_0927_2](https://user-images.githubusercontent.com/30659307/222313174-b37d5e7e-0f8c-4c09-bb6f-df186c6512f9.png)

**Reply by tjboldt** [reply]
The 3.3V line is used but pin 1 and 17 are tied together on my board and both should send 3.3V from the RPi to the chips on my board. Pin 3 is not connected and 2 and 4 are +5V as you noted. In theory it should be fine to cut 1 - 4 on your cable for your purposes but I haven’t tested that.
